CMakeLists.txt

```diff
@@ -154,6 +155,10 @@ if(OPENMC_ENABLE_COVERAGE)
   list(APPEND ldflags --coverage)
 endif()
 
+if(OPENMC_USE_MCPL)
+  list(APPEND ldflags -lmcpl)
+endif()
```

Comment:
I agree. But in the absence of a MCPLConfig.cmake-file we'd have to something like the above, no?

Reply:
Working on it!! https://github.com/mctools/mcpl/issues/56 :-)

Reply:
I think I gave the wrong link (https://github.com/mctools/mcpl/issues/58). Anyway, the point is that MCPL now exports the necessary cmake files for it to be found via find_package.

Give me a few more months and it might appear on conda as well, but I got a few other things lined up first.